src/tracks/playabletrack/wavetrack/ui/GetWaveDisplay.cpp

```diff
@@ -156,7 +156,10 @@ bool GetWaveDisplay(const Sequence &sequence,
          wxASSERT(false);
          // Do some defense against this case anyway
          while (pixel < nextPixel) {
-            min[pixel] = max[pixel] = rms[pixel] = 0;
+            columns[pixel].min = {};
+            columns[pixel].max = {};
+            columns[pixel].rms = {};
```

Comment:
I would write `columns[pixel] = {};` and leave it to https://en.cppreference.com/w/cpp/language/value_initialization